### PR TITLE
add gloo edge to integration

### PR DESCRIPTION
**Pull request by linsun**
Add gloo edge to integration page

### CODE REVIEW

File: data/companies.yml [hunk 1]
```diff
@@ -163,6 +163,12 @@ integrations:
     description: "Flagger is a Progressive Delivery Operator for Kubernetes."
     details:
       - "Flagger can run automated application analysis, testing, promotion and rollback for Canary (progressive traffic shifting), A/B Testing (HTTP headers and cookies traffic routing) and Blue/Green (traffic switching and mirroring) using Istio."
+  - name: "Gloo Edge"
+    logo: "/logos/gloo.png"
+    url: "https://www.solo.io/products/gloo-edge/"
+    description: "Gloo Edge is a Kubernetes native API gateway and ingress controller built on Envoy Proxy to facilitate and secure application traffic at the edge"
```
Comment (ericvn) [inline data/companies.yml:169]:
```suggestion
    description: "Gloo Edge is a Kubernetes native API gateway and ingress controller built on Envoy Proxy to facilitate and secure application traffic at the edge."
```

File: data/companies.yml [hunk 2]
```diff
@@ -163,6 +163,12 @@ integrations:
...
+    details:
+      - "Gloo Edge is a feature-rich, Kubernetes-native ingress controller, and next-generation API gateway. Gloo edge integrates nicely with Istio and services running in Istio mesh."
```
Comment (ericvn) [inline data/companies.yml:171]: We usually have Istio service mesh, so maybe add `service`:
```suggestion
      - "Gloo Edge is a feature-rich, Kubernetes-native ingress controller, and next-generation API gateway. Gloo edge integrates nicely with Istio and services running in Istio service mesh."
```

**Review by craigbox — APPROVED** [review]

**Comment by linsun** [comment]
Thanks @ericvn and @craigbox !

**Comment by craigbox** [comment]
1.13 will be branched for soon enough :)

**Pull request merged**